packages/vitetris.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Vitetris < Package
+  description 'Classic multiplayer tetris for the terminal'
+  homepage 'https://www.victornils.net/tetris/'
+  version '0.59.1'
+  license 'BSD-2-Clause'
+  compatibility 'all'
+  source_url 'https://github.com/vicgeralds/vitetris.git'
+  git_hashtag "v#{version}"
+  binary_compression 'tar.zst'
+
+  binary_sha256({
+    aarch64: '3794864d070a024d8b8595813e14dff1d810d0c2427d46680a6beaf051cb1ed2',
+     armv7l: '3794864d070a024d8b8595813e14dff1d810d0c2427d46680a6beaf051cb1ed2',
+       i686: '470209ea2b36b555411299355841b23a871fdd26b1d0b7d6d11b0c8057579e4e',
+     x86_64: '09cea01fcd5ba653b6a01bcf9d70712106c8a4dc5cfe4b0b8a152d8b0d717833'
+  })
+
+  depends_on 'curl' => :build
+  depends_on 'llvm' => :build
+  depends_on 'buildessential' => :build
+  depends_on 'glibc' # R
+
+  def self.build
+    system "./configure --prefix=#{CREW_PREFIX}"
+    system "make -s CFLAGS='-O2 -pipe -Wno-error=implicit-int -Wno-error=implicit-function-declaration'"
```

Comment:
```suggestion
    system "make CFLAGS='-Wno-error=implicit-int -Wno-error=implicit-function-declaration'"
```

Reply:
Did you test this before making the suggestion?

Reply:
I just know that `make -s` is silent, which we don't want, and that the rest of our CFLAGS already add `-O3 -pipe` (which is just a higher level of optimisations than -O2).

Reply:
Addressed.